feature/mpls/otg_tests/static_bgp_nexthop/static_bgp_nexthop_test.go

```diff
@@ -0,0 +1,447 @@
+	}
+	dutPort3 = attrs.Attributes{
+		Desc:    "DUT to ATE destination-3",
+		IPv4:    "192.0.2.9",
+		IPv6:    "2001:db8::9",
+		IPv4Len: ipv4PrefixLen,
+		IPv6Len: ipv6PrefixLen,
+	}
+	timeout  = 1 * time.Minute
+	interval = 20 * time.Second
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Test cases:
+//  1) Verify IPv4 MPLS forwarding.
+//		- Push the above DUT configuration.
+//		- Start traffic flow with MPLS[lbl-1000004] and IPv4 destined to IPV4-DST.
+//		- Verify that traffic arrives to ATE Port 2.
+//  2) Verify IPv6 MPLS forwarding.
+// 	`	- Push the above DUT configuration.
+// 		- Start traffic flow with MPLS[lbl-1000006] and IPv4 destined to IPV6-DST.
+// 		- Verify that traffic arrives to ATE Port 2.
+// 	3) Verify IPv4 traffic discard when BGP-NH is not available.
+// 		- Withdraw BGP-NH-V4 advertisement.
+// 		- Push the above DUT configuration.
+// 		- Start traffic flow with MPLS[lbl-1000004] and IPv4 destination set to IPV4-DST.
+// 		- Verify that traffic is discarded.
+// 	4) Verify IPv6 traffic discard when BGP-NH is not available.
+// 		- Withdraw BGP-NH-V6 advertisement.
+// 		- Push the above DUT configuration.
+// 		- Start traffic flow with MPLS[lbl-1000006] and IPv6 destination set to IPV6-DST.
+// 		- Verify that traffic is discarded.
+//  Details: https://github.com/openconfig/featureprofiles/blob/main/feature/mpls/otg_tests/static_bgp_nexthop/README.md
+//
+
+// TestMplsStaticLspBGPNextHop runs the verification steps
+func TestMplsStaticLspBGPNextHop(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	ate := ondatra.ATE(t, "ate")
+	configureDUT(t, dut)
+	cfgplugins.NewStaticMPLSLabel(t, dut, lspV4Name, mplsLabelV4, "", bgpNHv4, "ipv4")
+	cfgplugins.NewStaticMPLSLabel(t, dut, lspV6Name, mplsLabelV6, "", bgpNHv6, "ipv6")
+	topo := configureATE(t, ate)
+	verifyPortsUp(t, dut.Device)
+	checkBgpStatus(t, dut)
+	ate.OTG().StartTraffic(t)
+	time.Sleep(trafficDuration * time.Second)
+	ate.OTG().StopTraffic(t)
+	t.Run("Verify IPv4 MPLS forwarding", func(t *testing.T) {
+		if verifyFlowTraffic(t, ate, topo, ipv4Flow) {
+			t.Log("IPv4 Traffic MPLS forwarding Passed")
+		} else {
+			t.Error("IPv4 Traffic MPLS forwarding Failed")
+		}
+	})
+	t.Run("Verify IPv6 MPLS forwarding", func(t *testing.T) {
+		if verifyFlowTraffic(t, ate, topo, ipv6Flow) {
+			t.Log("IPv6 Traffic MPLS forwarding Passed")
+		} else {
+			t.Error("IPv6 Traffic MPLS forwarding Failed")
+		}
+	})
+	t.Run("Verify IPv4 traffic discard when BGP-NH-V4 is not available.", func(t *testing.T) {
+		withdrawBGPRoutes(t, []string{"BGP4.peer2.Route"})
+		verifyIpv4TrafficDiscard(t, ate, topo)
+	})
+	t.Run("Verify IPv6 traffic discard when BGP-NH-V6 is not available.", func(t *testing.T) {
+		withdrawBGPRoutes(t, []string{"BGP6.peer2.Route"})
+		verifyIpv6TrafficDiscard(t, ate, topo)
+	})
+}
+
+// configureDUT sets up the DUT interfaces, static LSPs, and BGP neighbors.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
```

Comment:
Also, where is the following from the Readme covered?

"Configure static routes on the DUT to discard traffic destined for BGP-NH-V4 and BGP-NH-V6. These routes should point to a Null0 with an administrative distance of 254 to ensure they are less preferred than the BGP routes. This prevents the DUT from using its IGP to reach the BGP next-hops.